commands/status.go

```diff
@@ -37,6 +38,9 @@ PMM-agent:
 	Connected : {{ .PMMAgentStatus.Connected }}{{ if .PMMAgentStatus.Connected }}
 	Time drift: {{ .PMMAgentStatus.ServerClockDrift }}
 	Latency   : {{ .PMMAgentStatus.ServerLatency }}
+
+PMM-admin:
```

Comment:
Please use `pmm-admin`, `pmm-agent`, `PMM Client`, `PMM Server` everywhere.
It should be `pmm-admin` there.